plugins/woocommerce/src/Admin/Features/Features.php

```diff
@@ -358,4 +370,26 @@ private function register_internal_class_aliases() {
 			class_alias( $new_class, $orig_class );
 		}
 	}
+
+	/**
+	 * Check if we're in an admin context where features should be loaded.
+	 *
+	 * @return boolean
+	 */
+	private static function should_load_features() {
```

Comment:
Random idea (not a blocker): might it be worth wrapping the return value of this method in a `woocommerce_load_admin_features` filter? Is there any scenario in which an extension might need these to be loaded even when not in an admin context?

Reply:
Yeah, I think that's a good idea.